hugegraph-core/src/main/java/com/baidu/hugegraph/backend/store/BackendStoreSystemInfo.java

```diff
@@ -64,17 +62,14 @@ public void init() {
 
     private Map<String, Object> info() {
         SchemaTransaction schema = this.graph.schemaTransaction();
-        PropertyKey pkey = null;
-        try {
-            pkey = schema.getPropertyKey(PK_BACKEND_INFO);
-        } catch (BackendException | IllegalStateException |
-                 UncheckedIOException ignored) {
-            // pass
-        }
```

Comment:
this BackendException | IllegalStateException | UncheckedIOException catch are for rocksdb, cassandra, hbase fetch without database&table